### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@vikivivi care to review

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -23739,6 +23755,11 @@ prehaps->perhaps
 preiod->period
 preivew->preview
 preivous->previous
+preject->project
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:23758]:
```suggestion
preject->project, prefect,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -23739,6 +23755,11 @@ prehaps->perhaps
...
+prejected->projected
+prejection->projection
+prejections->projections
+prejects->projects
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:23762]:
```suggestion
prejects->projects, prefects,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -20104,6 +20119,7 @@ mocule->module
 mocules->modules
 moddel->model
 moddels->models
+modelinng->modeling
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:20122]:
```suggestion
modelinng->modeling
modellinng->modelling
```
See https://www.grammarly.com/blog/modeling-or-modelling/

Do you want to add to the GB/US dict too?

**Review by peternewman — CHANGES_REQUESTED** [review]
A few suggestions

**Pull request merged**